javascript-forms/form-values.js

```diff
@@ -0,0 +1,18 @@
+var $contact = document.querySelector('#contact-form');
+var messageData = {
+  name: null,
+  email: null,
+  message: null
+};
```

Comment:
You should make this variable inside the function definition, it doesn't need to continue to exist after the function ends.